### ISSUE TITLE
Change SteamOS logo to Tux (mascot) in about page

### ISSUE THREAD

**Issue by barfin** [ValveSoftware/steam-for-linux#6507]
I think here is the most related section to ask this, sorry
since SteamOS has steam client built-in, it wouldn't make sense to make a download button for it in about page
it should be replaced with tux logo

**Comment by Myrddin-Wyllt** [comment]
Every Linux user that I know was a bit disappointed with the replacement of Tux with Steam logo. Since Valve doesn't seem to be actively promoting SteamOS, it certainly was an odd decision. Also, the SteamOS logo being identical to the Steam logo is certainly ambiguous.

Tux is recognizable to all Linux users. Please reconsider replacing Tux with the Steam/SteamOS logo in the Store listings as well.

**Comment by pruebahola** [comment]
Tux is more recognizable for all users so please valve, tux icon

**Comment by shigutso** [comment]
I totally agree, the Tux icon is much better than the ambiguous SteamOS icon.

**Comment by Weetile** [comment]
Agreed! The Steam logo is confusing for people who don't realise it means Linux/SteamOS.

**Comment by tunbridgep** [comment]
Agree.

**Comment by genX75** [comment]
Makes a lot of sense. This needs more exposure.

**Comment by TheTrueCoder** [comment]
I completely agree

**Comment by i-need-to-tell-you-something** [comment]
When Valve switched Tux icon for SteamOS back in 2015 some Linux gamers hated it, some liked it, some were neutral. No one should believe this thread that everyone hates it. If you want you can observe some varied opinions [here](https://www.gamingonlinux.com/2015/05/steam-replaces-the-linux-tux-logo-with-steamos/comment_id=37544), but please refrain from posting or liking that old thread to avoid necroposting. Warning: also contains unrelated arguing.

My opinion:
The previous tux icon looked hideous (as can be seen from [this image](https://i.imgur.com/PkMHF9E.png)) and the current SteamOS icon is confusing - looking at the about page I would assume that Steam is only supported on Windows and macOS and that this Steam icon on the download button would be there also if I were on Windows or macOS. I am neutral about the switch to SteamOS icon. I would like a good looking Tux icon or a SteamOS logo that is clearly differentiated from regular Steam logo. From Valve's perspective is preferable, except rebranding an existing icon is a lot of work.

**Comment by jobukkit** [comment]
This is a very long overdue change. Steam Machines barely exist anymore. The Steam Controller isn't even sold anymore.
Developers that still port their games to Linux are definitely not doing that for SteamOS, but for Linux laptops and desktops. 

How is an average Linux user supposed to understand that when the Steam logo appears in that specific position, it refers to Linux instead of Steam itself?
I actually saw a post lamenting the inability to play Dota 2 on a Linux laptop; he had to be informed that in fact you can. If Tux was used instead of the Steam logo (like in the beginning), it would be immediately understood. 

If Tux is too complex, use the Ubuntu or Debian logo. Not accurate but still far better than now.

**Comment by i-need-to-tell-you-something** [comment]
Btw for whomever didn't know this: 
A workaround exists in the add-on AugmentedSteam - an alternative tux icon feature. You can see a screenshot of the newest iteration [here](https://github.com/tfedor/AugmentedSteam/issues/561#issuecomment-704462710). Currently it is not displayed on Firefox due to [a bug](https://github.com/tfedor/AugmentedSteam/issues/893), but the fix is merged and awaiting next release (i.e. newer than v1.4.8).

**Comment by o-kotb** [comment]
It should be replaced with the GNU logo.

**Comment by jobukkit** [comment]
> It should be replaced with the GNU logo.

Steam can run on distro's with no GNU software outside of Glibc. Also, GNU hates 99% of games.

**Comment by pruebahola** [comment]
Tux icon should be the logo for Linux

**Comment by ZaxLofful** [comment]
It's kinda dumb to think that the Steam icon means Linux...Did no one test this out?

**Comment by paulspielmann** [comment]
+1

**Comment by nuggs** [comment]
Bring Tux back!

**Comment by droyd9** [comment]
+1, we love tux!

**Comment by EchoesForeAndAft** [comment]
I think the Steam icon means "Steam Play', not SteamOS, which I'm guessing includes the Steam Linux Runtime as well as Proton. Though using a Linux icon would undoubtedly be clearer.

**Comment by nosduco** [comment]
+1 for Tux!!

**Comment by Togradoty** [comment]
+1 for Tux, and i also think it would be easier for people newer to linux gaming to just see tux

**Comment by urbenlegend** [comment]
Yeah, I think the Steam logo is just confusing for a lot users when they look at platform support. Steam itself isn't an OS so it's odd that it's placed next to the Windows and Mac logos.

A tux icon would make things much more clear and promote the platform. Hell, even throw in a Steam Deck logo for Steam Deck verified games.

Valve, you've gotta be more proactive in promoting the OS you're doing such a wonderful job building and contributing to!!

**Comment by insanemal** [comment]
Mac and Windows get an apple and a Windows logo. 

Yet Linux gets a Steam logo? I'm sorry this doesn't make sense. 

A Tux logo would make far more sense. I'm sure there would be some licencing fees required. But long term it makes sense. 

Plus it raises visibility of Linux which is beneficial to Valve in the long run

**Comment by lou-dum-studio** [comment]
There is a simplyfied version of [tux](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b8/Other-linux-logo.svg/1280px-Other-linux-logo.svg.png).
Why wouldn't they use it instead of confusing steam logo?

**Comment by hazelnot** [comment]
Xenia is cooler than Tux but I do agree that it should more explicitly represent the fact that a Linux version is available, which to be fair Tux would do a better job at considering how well-known it is

**Comment by xAlpharax** [comment]
Definitely, please update the icons back to Linux Tuxies.

**Comment by LethalManBoob** [comment]
tux when :(

**Comment by pollux56** [comment]
tux when :(

**Comment by LeviLeal** [comment]
remember tux

**Comment by iodream** [comment]
> > tux when :(
> 
> Did a YouTuber or something get you guys here? This issue has been inactive for months and now suddenly many of you are asking for it

It was [posted](https://www.reddit.com/r/linux_gaming/s/q05NSPI8ML) to the Linux gaming subreddit.

For what it's worth I believe the situation is ok for now because Valve may want to distinguish the support as limited to their own distro. Even if in 90% cases the game will work on desktop Linux in general.

**Comment by etoven** [comment]
BiggiSize Upvote

**Comment by ZaxLofful** [comment]
> > > tux when :(
> > 
> > 
> > Did a YouTuber or something get you guys here? This issue has been inactive for months and now suddenly many of you are asking for it
> 
> It was [posted](https://www.reddit.com/r/linux_gaming/s/q05NSPI8ML) to the Linux gaming subreddit.
> 
> For what it's worth I believe the situation is ok for now because Valve may want to distinguish the support as limited to their own distro. Even if in 90% cases the game will work on desktop Linux in general.

Except they already support more than just their own OS, it's the same anywhere....Default Steam thing now, should reflect the real OS.

**Comment by Ceeee** [comment]
👍 +1

> > tux when :(
> 
> Did a YouTuber or something get you guys here? This issue has been inactive for months and now suddenly many of you are asking for it

Well this issue could have been resolved in that time span and now we are reminding the developers that it is due time. :-)
(It wasn't youtube but the steam forums that turned this topical again: https://steamcommunity.com/app/221410/discussions/0/4030222834114569815/ – The fact that this keeps resurfacing should prove the point.)

**Comment by z411** [comment]
With more people coming to Linux I think it's important to make them aware that the game has a native build for their platform. I've seen many new people and no one has been able to guess that the Steam logo means Linux.

**Comment by pruebahola** [comment]
Valve please, Tux should appear, together with steamOS logo if you wish but don't hide to TUX
